diff --git a/configure b/configure
index 78ddf3e..497ca43 100755
--- a/configure
+++ b/configure
@@ -6842,6 +6842,16 @@ $as_echo "#define GLIBC_2_20 1" >>confdefs.h
 	DEFAULT_SUPP="glibc-2.34567-NPTL-helgrind.supp ${DEFAULT_SUPP}"
 	DEFAULT_SUPP="glibc-2.X-drd.supp ${DEFAULT_SUPP}"
 	;;
+     2.21)
+	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: 2.21 family" >&5
+$as_echo "2.21 family" >&6; }
+
+$as_echo "#define GLIBC_2_21 1" >>confdefs.h
+
+	DEFAULT_SUPP="glibc-2.X.supp ${DEFAULT_SUPP}"
+	DEFAULT_SUPP="glibc-2.34567-NPTL-helgrind.supp ${DEFAULT_SUPP}"
+	DEFAULT_SUPP="glibc-2.X-drd.supp ${DEFAULT_SUPP}"
+	;;
      darwin)
 	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: Darwin" >&5
 $as_echo "Darwin" >&6; }
